superdesk/metadata/item.py

```diff
@@ -141,6 +141,15 @@
             }
         }
     },
+    'cvs': {
```

Comment:
if you are planing to have aggregations on `cvs` then it would be better to make it similar to `genre` or `anpa_category`. 
```
       'mapping': {
             'type': 'object',
             'properties': {
                 'qcode': not_analyzed,
                 'name': not_analyzed
             }
         }
```